### DISCUSSION TITLE
Integrate Revisions with Reusable Blocks

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Revisions#928] [Feature Requests]
> The WordPress core reusable blocks track revisions across changes, but I don't see a way to schedule a revision with PublishPress like we can with posts, pages, etc. Since we heavily rely on PublishPress Revisions for our workflow, this makes reusable blocks not very helpful for us.

**Comment by KratosGemini** [comment]
I believe this is a duplicate of #1118.
